docs/include/attributes-lsplugins.asciidoc

```diff
@@ -0,0 +1,8 @@
+/////
+ These settings control attributes in the LSR only. 
+They correspond to the VPR settings in logstash-docs/docs/versioned-plugins/include/attributes-ls-vpr.asciidoc
+When we update one, we must update settings in the other location
+/////
+
+:ecs-default:   Any value in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] for pipeline.ecs_compatibility other than disabled is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
```

Comment:
nit: quote value

```suggestion
:ecs-default:   Any value in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] for pipeline.ecs_compatibility other than `disabled` is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
```

Secondary: this setting can also be applied to an individual pipeline's definition in the `pipelines.yml`, and the BETA story is the same there.